### ISSUE TITLE
Space needed in Boundary accessible name

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/center-and-variability#658]
For https://github.com/phetsims/qa/issues/1268, there should be a space between "Boundary" and "A" or "B". Otherwise, it sounds odd with a screen reader. 

<img width="573" height="111" alt="Image" src="https://github.com/user-attachments/assets/17a98322-74a6-4753-8744-fcfb7953ef6e" />

<img width="235" height="242" alt="Image" src="https://github.com/user-attachments/assets/040df1ed-9747-4b91-a9d2-ae504fbf6cf6" />

**Comment by marlitas** [comment]
Done above. Ready to cherry pick.

**Comment by Nancy-Salpepi** [comment]
Sounds good in rc.2.
Closing.
